### ISSUE TITLE
Question: Does trimesh do any kind of transformation on coordinates when loading an stl

### ISSUE THREAD

**Issue by Benjamin-Fouquet** [mikedh/trimesh#1310]
Hello,

When loading a stl file in world coordinate from 3D slicer, I do not find "back" the same points when accessing mesh.vertices. It seems there is some kind of coordinate transformation that I fail to understand, could you help me on that ?

Additional question, is there a way to ensure I stay in world coordinate system using trimesh?

**Comment by mikedh** [comment]
Yeah by default it merges vertices and removes NaN/inf values, you can turn that it off by loading with `mesh = trimesh.load(file_name, process=False)`.

With regard to 'world coordinates' STL's don't have transforms so it should just be an array of points unless you create a `trimesh.Scene` to hold the mesh.

**Comment by whateverforever** [comment]
Currently having the same issue. Blender and meshlab agree on the vertex ordering, but trimesh has a different one. It seems blender and meshlab go vertex by vertex in the file, only loading a new one, if it hasn't already been loaded. trimesh seems to load all vertices as-is (i.e. almost all of them are duplicates) and merges them later on in a second step (losing the ordering). Did I get that correctly @mikedh ?
